Track Buddy/New Group/AccelerometerView.swift

```diff
@@ -67,6 +68,17 @@ struct AccelerometerView: View {
             .foregroundColor(.orange)
     }
     
+    private var resetButton: some View {
+        Button {
+            motionManager.resetMaxValues()
+        } label: {
+            Text("Reset")
+        }
+        .padding()
+        .background(RoundedRectangle(cornerRadius: 20.0))
```

Comment:
Magic number should be in a Metrics enum in this view 
(don't need that "drawing constants" comment for all of these btw)